pixman/plan.sh

```diff
@@ -0,0 +1,35 @@
+pkg_origin=core
+pkg_name=pixman
+pkg_description="A low-level software library for pixel manipulation"
+pkg_version=0.34.0
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("MIT")
+pkg_source="https://www.cairographics.org/releases/pixman-${pkg_version}.tar.gz"
+pkg_shasum=21b6b249b51c6800dc9553b65106e1e37d0e25df942c90531d4c3997aa20a88e
+pkg_deps=(core/glibc core/gcc-libs core/libpng core/zlib)
+pkg_build_deps=(core/gcc core/make core/pkg-config core/diffutils core/file)
+pkg_include_dirs=(include)
+pkg_lib_dirs=(lib)
+
+do_prepare() {
+    if [[ ! -r /usr/bin/file ]]; then
+        ln -sv "$(pkg_path_for file)/bin/file" /usr/bin/file
+        _clean_file=true
+    fi
+}
+
+do_build() {
+    export PKG_CONFIG_PATH="$(pkg_path_for libpng)/lib/pkgconfig"
```

Comment:
This callback can be removed post-https://github.com/habitat-sh/core-plans/pull/40